### ISSUE TITLE
`elm-pages --version` prints incorrect version number

### ISSUE THREAD

**Issue by adamdicarlo0** [dillonkearns/elm-pages#405]
Version 3.0.6 reports itself as 3.0.3.

```
❯ npm init
[...]
Is this OK? (yes) yes

~/work/project
❯ npm add elm-pages@latest
[...]

~/work/project
❯ ./node_modules/.bin/elm-pages --version                     
3.0.3

~/work/project
❯ cat ./node_modules/elm-pages/package.json| jq .version 
"3.0.6"
```

`npx elm-pages init <folder>` created a project with elm-pages@3.0.3 (a bit out of date) -- perhaps that is related?

**Comment by dillonkearns** [comment]
Fixed in 3.0.7, thank you!

**Comment by Erudition** [comment]
I'm having the same issue with 3.0.12, which prints 3.0.11.
